client/state/test/index.js

```diff
@@ -0,0 +1,47 @@
+require( 'lib/react-test-env-setup' )();
+/**
+ * External dependencies
+ */
+import { expect } from 'chai';
+import mockery from 'mockery';
+
+describe( 'state', () => {
+	var createReduxStore;
+	before( () => {
+		mockery.registerMock( 'lib/themes/middlewares.js', {
```

Comment:
Yeah, I noticed that and was a little confused. But I thought it was a defensive move in case reducers were added that made use of the middleware. Probably should have mentioned it.